time stamp
int32 data_length
float64 total_voltage
float64 current
float64 residual_capacity
float64 nominal_capacity
int32 cycle_life
int32 product_date_year
int32 product_date_month
int32 product_date_day
uint16 balance_status
uint16 balance_status_high
uint16 protection_status
float64 version
int32 rsoc
uint8 control_status
int32 num_cell
int32 num_ntc
float64 temp1
float64 temp2
uint16 check_sum
